demos/0_intro.py

```diff
@@ -187,17 +184,16 @@ def __call__(self, x):
 delta = 1e-4
 n_samples = 4
 
-key, subkey = random.split(key)
-pos_init = jft.Vector(jft.random_like(subkey, signal_response.domain))
+key, k_i, k_o = random.split(key, 3)
 # NOTE, changing the number of samples always triggers a resampling even if
 # `resamples=False`, as more samples have to be drawn that did not exist before.
 samples, state = jft.optimize_kl(
     nll,
-    pos_init,
+    jft.Vector(jft.random_like(k_i, signal_response.domain)),
```

Comment:
Shall we advertise `signal_response.init` here instead?

Reply:
yes, we absolutely should! Thanks! :)